hdr_html_canvas_element.md

```diff
@@ -162,6 +162,10 @@ not quantized, i.e. R' = 0.5 is represented by the floating point value 0.5.
 _NOTE: `dataType = "unorm8"` corresponds to HTML Canvas as it exists today and
 should not be used to represent HDR signal, as detailed in the introduction._
 
+The `HTMLCanvasElement` methods `toDataURL()` and `toBlob()` should produce
+resources that best match the fidelity of the underlying HTML Canvas element
+(subject to the limitations of the implementation and the requested format).
```

Comment:
That seems a bit vague, hand-wavy and hard to test.

It seems better to crisply state that the color space of the 2D context should be preserved in `toDataURL()` and `toBlob()`. (And yes, if the requested format can't do that then there is an issue, but all common formats can represent that information).